test/cards/AdaptiveCardStyles.xaml

```diff
@@ -0,0 +1,305 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
@dandriscoll Will consolidate this folder to remove unnecessary files. Currently, this is just showing I copied all the files from AdaptiveCards folder.